mcstatus/__init__.py

```diff
@@ -1,4 +1,4 @@
-from mcstatus.server import BedrockServer, JavaServer, MCServer  # noqa: F401
+from mcstatus.server import BedrockServer, JavaServer, MCServer
```

Comment:
Ruff is able to pick up that this is suposed to be "re-exported", as it's present in `__all__`